### DISCUSSION TITLE
Unable to access airsdk.dev?

### DISCUSSION THREAD

**Discussion by ylazy** [airsdk/Adobe-Runtime-Support#3464] [General]
Hi!

I'm unable to access [airsdk.dev](https://airsdk.dev/) from this morning til now using my pc (but it's ok with my phone)

![image](https://github.com/user-attachments/assets/1249dab0-e7e5-4928-91ab-83504a8e4ab0)

I've tried Incognito Mode but still the same.

Am I the only one?

Thanks!

**Comment by marchbold** [comment]
Working fine for me. Our host did have an issue this morning (GMT+10) though so may have been offline for a while then.
